QUEST.tsv

```diff
@@ -2535,7 +2535,7 @@ QUEST_20151001_002542
 QUEST_20151001_002543	
 QUEST_20151001_002544	
 QUEST_20151001_002545	
-QUEST_20151001_002546	
+QUEST_20151001_002546	$Medis Diena was not Nature's will. After all, Nature is governed by the Goddess, and she would never do harm to us all.
```

Comment:
Just thought the Druid Master might refer to nature as an entity. Reverting it to lower case then.